custommetric.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	monitoring "cloud.google.com/go/monitoring/apiv3"
+	"cloud.google.com/go/monitoring/apiv3/v2/monitoringpb"
+	timestamp "github.com/golang/protobuf/ptypes/timestamp"
+	"google.golang.org/genproto/googleapis/api/label"
+	"google.golang.org/genproto/googleapis/api/metric"
+	metricpb "google.golang.org/genproto/googleapis/api/metric"
+	monitoredres "google.golang.org/genproto/googleapis/api/monitoredres"
+)
+
+// [START monitoring_write_timeseries]
```

Comment:
the comment is not necessary. Is this like generated code?

Reply:
removed